GoogleForCreators/web-stories-wp
# Dashboard: My Stories View Karma Tests

## Summary

Add Karma integration tests for My Stories view and continue stubbing out `ApiProviderFixture`. Also adds an initial test for navigation to Explore Templates


## User-facing changes

None

## Testing Instructions

1. Validate that the GitHub `Dashboard Integration Tests` action/check is successful for this PR as it will run all of the tests headlessly
2. Pull this branch and run `npm run test:karma:dashboard` and see that tests complete successfully

---


#1998
